collect_app/src/main/java/org/odk/collect/android/utilities/AuthDialogUtility.java

```diff
@@ -37,6 +39,12 @@
 
     private EditText username;
     private EditText password;
+    private GeneralSharedPreferences generalSharedPreferences;
```

Comment:
Why not use `@Inject` here instead of the constructor?

Reply:
Done